pytorch_lightning/trainer/trainer.py

```diff
@@ -296,6 +296,7 @@ def __init__(
         self._device_type = DeviceType.CPU
         self._distrib_type = None
         self._running_stage = None
+        self.is_predicting = False
```

Comment:
is it setter? I thought we discussed having it as a read-only property

Reply:
It is limitation of the current routing system. I will take care of this in another PR.